### ISSUE TITLE
Not working in release mode with new arch enabled (72.6)

### ISSUE THREAD

**Issue by JohnathanWhite** [ammarahm-ed/react-native-actions-sheet#317]
Works in debug but with a release build (w/ new arch) the sheet is not visible.

**Comment by mauricedoepke** [comment]
I was able to solve this issue for me.
I noticed, that directly using the ActionsSheets work, only usage via the SheetManager was broken.

With a bit more debugging, I noticed, that the file with all my registerSheet functions was not imported anywhere and thus the sheets never actiolly got registered. For some reason this file was stil run in dev mode and also in expo sdk 47 release mode.

So I suspect it was most likely not event react native, but a change in the metro bundler, that caused this issue for me.

TLDR: I forgot to register the actions sheets.

Does that fix it for you as well @JohnathanWhite ?

**Comment by OFD16** [comment]
I have the same problem, the react-native-modal package does not work in the same release mode (apk) as the react-native-actions-sheets package. I think it's a general problem that I can see.

How can we solve this problem, I opened a new topic about it ([issue381](https://github.com/ammarahm-ed/react-native-actions-sheet/issues/381))

- react-native-actions-sheets 
- react-native-modal

those packages have problem in android release mode (apk) render problem. They are not working as in issue you can check.
`android > gradle.properties`
`newArchEnabled=true`
